rero_ils/modules/api.py

```diff
@@ -95,19 +95,50 @@ def bulk_index(self, record_id_iterator, doc_type=None):
         """
         self._bulk_op(record_id_iterator, op_type='index', doc_type=doc_type)
 
-    def _index_action(self, payload):
-        """Bulk index action.
-
-        :param payload: Decoded message body.
-        :returns: Dictionary defining an Elasticsearch bulk 'index' action.
-        """
+    def get_record_class(self, payload):
```

Comment:
this can be probably private with a _ prefix

Reply:
ok